wp/resources/sample_binaries/retrowrite_stub/main_bil.txt

```diff
@@ -0,0 +1,84 @@
+
```

Comment:
Can we remove this file from VC?

Reply:
Yeah. I'll get rid of all of these arbitrary small changes.